docs/repositories-configure/using-submodules.md

```diff
@@ -4,35 +4,50 @@
 
 By default, Codacy does normal Git clones that **don't include submodules** to ensure that we only clone necessary repositories. If your organization needs to use submodules, you can request Codacy to enable this feature for you.
 
+!!! note
+    On GitHub organizations, your repository and the repositories that you add as submodules must belong to the same GitHub organization.
+
 ## Prerequisites for using submodules
 
 1.  Contact us at <mailto:support@codacy.com> asking to enable submodules on Codacy.
 
 1.  **If you're using Codacy Self-hosted**, [update your license](../chart/maintenance/license.md).
 
-1.  If your submodules are:
+1.  If your submodules are:<!--TODO PLUTO-772 Validate prerequisites-->
```

Comment:
@andrzej-janczak Now using installation access tokens, do these prerequisites related to the repository Git URL still apply to GitHub? 🤔

Reply:
For github there will be no SSH at all 

Thus for both private and public for Github it should use https